src/core/saca-hub/variables.tf

```diff
@@ -140,3 +144,112 @@ variable "bastion_ipconfig_name" {
   default     = "mlzDemoBastionHostIpCfg"
   type        = string
 }
+
+#################################
+# Jumpbox VM Configuration
+#################################
+
+variable "jumpbox_subnet" {
+  description = "The subnet for jumpboxes"
+  type = object({
+    name              = string
+    address_prefixes  = list(string)
+    service_endpoints = list(string)
+
+    enforce_private_link_endpoint_network_policies = bool
+    enforce_private_link_service_network_policies  = bool
+
+    nsg_name = string
+    nsg_rules = map(object({
+      name                       = string
+      priority                   = string
+      direction                  = string
+      access                     = string
+      protocol                   = string
+      source_port_range          = string
+      destination_port_range     = string
+      source_address_prefix      = string
+      destination_address_prefix = string
+    }))
+
+    routetable_name = string
+  })
+  default = {
+    name              = "mlzDemoJumpboxSubnet"
+    address_prefixes  = ["10.0.100.160/27"]
+    service_endpoints = ["Microsoft.Storage"]
+
+    enforce_private_link_endpoint_network_policies = false
+    enforce_private_link_service_network_policies  = false
+
+    nsg_name = "mlzDemoJumpboxSubnetNsg"
+    nsg_rules = {
+      "allow_ssh" = {
+        name                       = "allow_ssh"
+        priority                   = "100"
+        direction                  = "Inbound"
+        access                     = "Allow"
+        protocol                   = "Tcp"
+        source_port_range          = "22"
+        destination_port_range     = ""
+        source_address_prefix      = "*"
+        destination_address_prefix = ""
+      },
+      "allow_rdp" = {
+        name                       = "allow_rdp"
+        priority                   = "200"
+        direction                  = "Inbound"
+        access                     = "Allow"
+        protocol                   = "Tcp"
+        source_port_range          = "3389"
+        destination_port_range     = ""
+        source_address_prefix      = "*"
+        destination_address_prefix = ""
+      }
+    }
+
+    routetable_name = "mlzDemoJumpboxSubnetRt"
+  }
+}
+
+variable "jumpbox_keyvault_name" {
+  description = "The name of the jumpbox virtual machine keyvault"
+  default     = "mlzDemoJumpboxVmKv"
```

Comment:
My deployment got this error, which I think is related to this value.

```
Error: creating Vault: (Name "mlzDemoJumpboxVmKv" / Resource Group "rg-saca-jbh003"): keyvault.VaultsClient#CreateOrUpdate: Failure sending request: StatusCode=0 -- Original Error: autorest/azure: Service returned an error. Status=<nil> Code="VaultAlreadyExists" Message="The vault name 'mlzDemoJumpboxVmKv' is already in use. Vault names are globaly unique so it is possible that the name is already taken. If you are sure that the vault name was not taken then it is possible that a vault with the same name was recently deleted but not purged after being placed in a recoverable state. If the vault is in a recoverable state then the vault will need to be purged before reusing the name. For more information on soft delete and purging a vault follow this link https://go.microsoft.com/fwlink/?linkid=2147740."
```

Reply:
Ah. KeyVaults are a global resource. My mistake. Will generate a name.